### PR TITLE
Implemented "Model 1" from Brimkov et al. for zero forcing problem on Simple Graph

### PR DESCRIPTION
**Pull request by joshyaffee**
… simple graph

`compute` method implemented for SimpleGraph, previous method is commented out... what should we do with this?

`_compute` method implemented for DiGraph, called by `compute`

# Description

### Documentation and Style

- [1/2] I have added docstrings to any new functions or types, as applicable.
- [X] My code follows the project's coding style and guidelines.
- [ ] TODO: If you have added a new algorithm or function, you have also included corresponding unit tests.
- [X] I confirm that any code added is my own work and is not copied from other sources.
- [X] I have requested a review from at least one other contributor.

### Developer's Declaration:

By submitting this Pull Request, I confirm the following:

- [X] The code I am submitting is my own work and I have the rights to use and distribute it.
- [X] I confirm that the code I have added is not copied from other sources and that I have appropriately cited or acknowledged any external code or references used.
- [X] I understand that my code will be reviewed and any changes requested must be addressed.

### PR Type

What kind of change does this PR introduce?
- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation content changes
- [ ] Unit Testing
- [ ] Other... Please describe:

### CODE REVIEW

File: src/Invariants/zero_forcing/minimum_zero_forcing_set.jl [hunk 1]
```diff
@@ -3,21 +3,114 @@
 
 Return a minimum zero forcing set of `g`.
 
+"""
+# previous implementation: what should we do with this?
+
+# function compute(
+#     ::Type{MinimumZeroForcingSet},
```
Comment (RandyRDavila) [inline src/Invariants/zero_forcing/minimum_zero_forcing_set.jl:10]: Probably keep it with a new method for `::BruteForceMinimumZeroForcingSet`

File: src/Invariants/zero_forcing/minimum_zero_forcing_set.jl [hunk 2]
```diff
@@ -3,21 +3,114 @@
...
 
-    # Get the number of vertices in `g`.
+    # convert to directed graph, replacing each edge with two directed edges
+    directed_g = Graphs.DiGraph(g)
+
+    return _compute(MinimumZeroForcingSet, directed_g)
+end
+
+# helper function for computing minimum zero forcing set of undirected graph...
+# if it is already directed, can we just use this function? (and rename + document it)
```
Comment (RandyRDavila) [inline src/Invariants/zero_forcing/minimum_zero_forcing_set.jl:46]: Yeah, I think so

Comment (joshyaffee) [reply src/Invariants/zero_forcing/minimum_zero_forcing_set.jl:46]: I'll address this comment and the previous one in my next commit. I wanted to push the bug fixes first before changing anything else.

**Comment by RandyRDavila** [comment]
LGTM!

**Review by RandyRDavila — APPROVED** [review]

**Pull request merged**